app-gen.py

```diff
@@ -369,8 +369,9 @@ def _gen_fluxcd_manifest(self):
 
 
             # generate a manifest/static-overrides file
-            system_override_file = flux_dir + '/' + a_chart['name'] + '/' + a_chart['name'] + '-static-overrides.yaml'
-            open(system_override_file, 'w').close()
+            static_override_file = flux_dir + '/' + a_chart['name'] + '/' + a_chart['name'] + '-static-overrides.yaml'
+            with open(static_override_file, 'w') as f:
+                f.write('image:\n  repository: docker.io/brunomuniz/poc-starlingx')
```

Comment:
why is this here?

Reply:
I suggested as a placeholder until we figure this out for good. It lacks the `# Placeholder.` comment that I suggested, but that's the idea: to be a placeholder so the generated package "kinda" works.

Reply:
But why put something related to a specific application in a generator that is to be used with any StarlingX application?

Reply:
Ok, I'm convinced, there's not much value in the placeholder, let's work to make the generator able to take care of this automatically anyway.